sdk/scheduler/src/main/java/com/mesosphere/sdk/specification/DefaultTaskSpec.java

```diff
@@ -44,6 +51,14 @@
     @Valid
     private final Collection<ConfigFileSpec> configFiles;
 
+    @DecimalMin("0")
+    @DecimalMax("1209600") //<< two weeks
```

Comment:
This max of two weeks is from the product specification, so yes a reasonable max.